### DISCUSSION TITLE
Use supabase cli with docker?

### DISCUSSION THREAD

**Discussion by magick93** [supabase/supabase#8507] [Questions]
Hello

I've been using the supabase cli and it does make migration and version controlling of schema changes really easy. However we are using the supabase docker-compose.yml setup.

I'm wondering if it is possible to use the supabase cli to do schema migrations using docker rather than `supabase start`?

**Comment by fabianuribe** [comment]
Looks like it was introduced on the latest docker for desktop on Mac. 

See temp solution here: https://github.com/docker/for-mac/issues/6531#issuecomment-1288237259
